lang/guile/Portfile

```diff
@@ -51,66 +26,92 @@ long_description    \
         threads, dynamic linking, a foreign function call \
         interface, powerful string processing, and many other \
         features needed for programming in the real world.
-
+        
 distname            guile-${version}
 homepage            https://www.gnu.org/software/guile/guile.html
 master_sites        gnu
+use_xz              yes
+
+checksums           rmd160  50534bf521961d08fb1b4fb6e1d5dbefe305a4f3 \
+                    sha256  1a2625ac72b2366e95792f3fe758fd2df775b4044a90a4a9787326e66c0d750d \
+                    size    5750636
+
+compiler.c_standard 2011
 
-depends_lib         port:readline \
+# https://trac.macports.org/ticket/59365
+xcode_workaround.fixed_xcode_version 11.2
+xcode_workaround.type avoid_xcode_compiler
+
+depends_build-append \
+                    port:autoconf \
+                    port:automake \
                     port:gettext \
+                    port:texinfo
+
+# guile-config is a wrapper around pkg-config
+depends_lib-append  port:boehmgc \
+                    port:gettext-runtime \
+                    port:gmp \
+                    port:libffi \
                     port:libiconv \
                     port:libtool \
-                    port:gmp \
                     port:libunistring \
-                    port:boehmgc \
-                    port:libffi \
-                    port:ncurses
-
-# guile-config is a wrapper around pkg-config
-depends_lib-append  port:pkgconfig
+                    port:ncurses \
+                    port:pkgconfig \
+                    port:readline
 
 # fixes: sed: -i may not be used with stdin
 depends_build-append port:gsed
 configure.env-append SED=${prefix}/bin/gsed
-build.env-append    SED=${prefix}/bin/gsed
+build.env-append     SED=${prefix}/bin/gsed
 
 configure.args      CPPFLAGS="-I${prefix}/include" \
                     LDFLAGS="-L${prefix}/lib" \
                     --infodir="${prefix}/share/info" \
                     --mandir="${prefix}/share/man" \
                     --enable-regex \
                     --disable-error-on-warning \
-                    --disable-silent-rules
-
-# Unable to cross compile, so we need to be able to run the built code
-if {${os.arch} eq "i386" && ${os.major} >= 11} {
-    supported_archs i386 x86_64
-    set universal_archs_supported {i386 x86_64}
-} elseif {${os.arch} eq "i386" && ${build_arch} eq "x86_64"} {
-    supported_archs i386 x86_64 ppc
-    set universal_archs_supported {i386 x86_64 ppc}
-} elseif {${os.arch} eq "i386"} {
-    supported_archs i386 ppc
-    set universal_archs_supported {i386 ppc}
-} elseif {${build_arch} eq "ppc64"} {
-    supported_archs ppc ppc64
-    set universal_archs_supported {ppc ppc64}
-} else {
-    supported_archs ${build_arch}
-    set universal_archs_supported ${build_arch}
+                    --disable-silent-rules \
+                    --enable-shared
```

Comment:
As with a dep list, alphabetize these as well. (Though `CPPFLAGS` and `LDDFLAGS` are fine to keep at the top of the list.)